spec/unit/util/package/version/rpm_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+require 'puppet/util/package/version/rpm'
+
+describe Puppet::Util::Package::Version::Rpm do
+
+  context "when parsing an invalid version" do
+    it "if is parsing symbols" do
+      expect { described_class.parse(:absent)}.to raise_error
```

Comment:
please specify which error must be raised, otherwise this might generate false positives